### PR TITLE
8319238: JMX ThreadPoolAccTest.java is too verbose and should fail before timeout

### PR DESCRIPTION
**Pull request by kevinjwalls**
Discovered while testing changes that made this test fail.  The test failure is hard to diagnose as it logs and retries at full speed, possibly forever, until timeout.  This can hit a log file limit.  We can save thousands of lines of text being printed when the test runs normally and successfully, by waiting half a second before doing the Principal-checking which is the purpose of the test.

---------
### Progress
- [x] Change must be properly reviewed (1 review required, with at least 1 [Reviewer](https://openjdk.org/bylaws#reviewer))
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue

### Issue
 * [JDK-8319238](https://bugs.openjdk.org/browse/JDK-8319238): JMX ThreadPoolAccTest.java is too verbose and should fail before timeout (**Bug** - P4)

### Reviewers
 * [Serguei Spitsyn](https://openjdk.org/census#sspitsyn) (@sspitsyn - **Reviewer**)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.org/jdk.git pull/16456/head:pull/16456` \
`$ git checkout pull/16456`

Update a local copy of the PR: \
`$ git checkout pull/16456` \
`$ git pull https://git.openjdk.org/jdk.git pull/16456/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 16456`

View PR using the GUI difftool: \
`$ git pr show -t 16456`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.org/jdk/pull/16456.diff">https://git.openjdk.org/jdk/pull/16456.diff</a>

</details>

### Webrev
[Link to Webrev Comment](https://git.openjdk.org/jdk/pull/16456#issuecomment-1789432935)

### CODE REVIEW

File: test/jdk/javax/management/monitor/ThreadPoolAccTest.java [hunk 1]
```diff
@@ -151,6 +163,9 @@ private static boolean testPrincipals(ObservedObject[] monitored, ObjectName[] m
         for (int i = 0; i < 6; i++) {
             String principal =  monitored[i].principal;
             String expected = principals[i / 3];
+
+            echo("testPrincipals: monitored: " + monitored[i] + " principal: " + principal + " expected: " + expected);
```
Comment (sspitsyn) [inline test/jdk/javax/management/monitor/ThreadPoolAccTest.java:167]: With this in place, should printing of principal at line 176 be removed?

**Review by sspitsyn — APPROVED** [review]
Looks reasonable.
Thanks,
Serguei

File: test/jdk/javax/management/monitor/ThreadPoolAccTest.java (hunk 1 above)
Comment (kevinjwalls) [reply test/jdk/javax/management/monitor/ThreadPoolAccTest.java:167]: Right, it is duplicated.  That new line is useful, I really wanted to see something before we check for null.
Then we have the existing printing before we check further, which I did not disturb.  I'm sure it's not a big deal either way.

Comment (sspitsyn) [reply test/jdk/javax/management/monitor/ThreadPoolAccTest.java:167]: Up to you.

**Pull request closed without merging**